README.md

```diff
@@ -1,23 +1,22 @@
-# Replit Extensions API Client
+# Replit Extensions Client
 
-The Replit Extensions client is a module that allows you to easily interact with the Workspace
+A monorepo containing all NPM packages related to Replit Extensions.
 
-- Repositories
-  - https://github.com/replit/extensions
-  - https://github.com/replit/extensions-react
 - NPM Packages
   - https://www.npmjs.com/package/@replit/extensions
   - https://www.npmjs.com/package/@replit/extensions-react
-- [GitHub Repository](https://github.com/replit/extensions)
+- [Repository](https://github.com/replit/extensions)
 - [Documentation](https://docs.replit.com/extensions)
-   - [Resources](https://docs.replit.com/extensions/resources)
-   - [API Modules](https://docs.replit.com/extensions/category/api-reference)
-   - [React Client](https://docs.replit.com/extensions/category/react)
-- [Ask Forum](https://ask.replit.com/c/extensions)
+  - [Resources](https://docs.replit.com/extensions/resources)
+  - [API Modules](https://docs.replit.com/extensions/category/api-reference)
+  - [React Client](https://docs.replit.com/extensions/category/react)
+- [Discourse Category](https://ask.replit.com/c/extensions)
 - [React Extension Template](https://replit.com/@replit/React-Extension?v=1)
 - [HTML/CSS/JS Extension Template](https://replit.com/@replit/HTMLCSSJS-Extension?v=1)
 
-[![Run on Replit button](https://user-images.githubusercontent.com/50180265/228865994-ccf7348e-ffb7-454e-bc4e-ce90df6c09bc.png)](https://replit.com/github/replit/extensions)
-
-## Help
-If you don't understand something in the documentation, have found a bug, or would like to request a feature, you can get help on the [Ask Forum](https://ask.replit.com/c/extensions).
+## Developer Guide
```

Comment:
I guess each of the packages should have their own developer guides

Reply:
I'll write some more detailed stuff on it